Default.sublime-keymap

```diff
@@ -308,6 +317,15 @@
         ]
     },
     {
+        "keys": ["k"],
+        "command": "gs_status_navigate_file",
+        "args": { "forward": false },
+        "context": [
+            { "key": "setting.git_savvy.status_view", "operator": "equal", "operand": true },
+            { "key": "setting.git_savvy.vintageous_friendly", "operator": "equal", "operand": true }
+        ]
+    },
+    {
```

Comment:
It does, can be overriden with `"context": [{ "key": "vi_command_mode_aware" }]`. As I mentioned, this commit was made as an idea. I'll handle command mode properly now, since I know  that you're willing to merge vintagoues support.